util/sfc_climo_gen/run.wcoss2.sh

```diff
@@ -22,88 +23,6 @@ export APRUN_SFC="mpiexec -n 24 -ppn 24 -cpu-bind core"
 
 export BASE_DIR=$PBS_O_WORKDIR/../..
 
-source ${BASE_DIR}/sorc/machine-setup.sh > /dev/null 2>&1
-module use ${BASE_DIR}/modulefiles
-module load build.$target.intel
-module list
-
-#-------------------------------------
-# Set model resolution.
-#-------------------------------------
-
-export res=384
-
-#-------------------------------------
-# Where the model "grid", "mosaic" and "oro" files reside.
-#-------------------------------------
-
-export FIX_FV3=${BASE_DIR}/fix/orog/C${res}
-
-#-------------------------------------
-# Uncomment for regional grids.
-#-------------------------------------
-
-##HALO=3
-##export GRIDTYPE=regional
-
-#-------------------------------------------------------------
-# Choose which soil type and vegetation type data to use.
-#
-# For viirs-based vegetation type data, set to:
-#   1) "viirs.igbp.0.1" for global 0.10-deg data
-#   2) "viirs.igbp.0.05" for global 0.05-deg data
-#   3) "viirs.igbp.0.03" for global 0.03-deg data
-#   4) "viirs.igbp.conus.30s" for CONUS 30s data
-#   4) "viirs.igbp.nh.30s" for NH 30s data
-#   4) "viirs.igbp.30s" for global 30s data
-#
-# For the modis-based vegetation data, set to:
-#   1) "modis.igbp.0.05" for global 0.05-deg data
-#   2) "modis.igbp.0.03" for global 0.03-deg data
-#   3) "modis.igbp.conus.30s" for CONUS 30s data
-#   4) "modis.igbp.nh.30s" for NH 30s data
-#   5) "modis.igbp.30s" for global 30s data
-#
-# For STATSGO soil type data
-#   1) "statsgo.0.05" for global 0.05-deg data
-#   2) "statsgo.0.03" for global 0.03-deg data
-#   3) "statsgo.conus.30s" for CONUS 30s data
-#   4) "statsgo.nh.30s" for NH 30s data
-#   5) "statsgo.30s" for global 30s data
-#
-# For Beijing Norm. Univ. soil type data
-#   1) "bnu.30s" for global 30s data
-#-------------------------------------------------------------
-
-export veg_type_src="modis.igbp.0.05"
-
-export soil_type_src="statsgo.0.05"
-
-#-------------------------------------
-# Set working directory and directory where output files will be saved.
-#-------------------------------------
-
-export WORK_DIR=/lfs/h2/emc/stmp/$LOGNAME/work.sfc
-export SAVE_DIR=/lfs/h2/emc/stmp/$LOGNAME/sfc.C${res}
-
-#-------------------------------------
-# Should not have to touch anything below here.
-#-------------------------------------
-
-if [[ $GRIDTYPE = "regional" ]]; then
-  HALO=$(( $HALO + 1 ))
-  export HALO
-  ln -fs $FIX_FV3/C${res}_grid.tile7.halo${HALO}.nc $FIX_FV3/C${res}_grid.tile7.nc
-  ln -fs $FIX_FV3/C${res}_oro_data.tile7.halo${HALO}.nc $FIX_FV3/C${res}_oro_data.tile7.nc
-fi
-
-export input_sfc_climo_dir=${BASE_DIR}/fix/sfc_climo
-
-ulimit -a
-ulimit -s unlimited
-
-rm -fr $WORK_DIR $SAVE_DIR
-
-${BASE_DIR}/ush/sfc_climo_gen.sh
+$PBS_O_WORKDIR/sfc_gen.sh
```

Comment:
Why was the block of code below in the original file removed?  I assume we want to keep this functionality on WCOSS2, correct?

Reply:
The common logic - including the invocation of `sfc_climo_gen.sh` - was moved to new script `sfc_gen.sh`.  That new script is called from the hera and wcoss2 (and any future) driver scripts.

Reply:
Sorry, I got confused when comparing the wrong files.  Looks good!